### PR TITLE
PROD-73: created layout page

### PR DESCRIPTION
**Pull request by gnaizer**
# Reviewers

- [ ] @vikabelous
- [ ] @omaik  
- [x] @Pavliv
- [x] @imysheniuk

# Jira task

http://ssu-jira.softserveinc.com/browse/PROD-73

# Description

Have to create layout page, using Bootstrap and Haml

# Checklist:

- [ ] Rubocop doesn't raise any warnings and offenses
screenshot from 2018-04-05 16-27-09

- [ ] All specs pass
- [ ] New features are covered with new specs
- [ ] PR is approved by 2 team mates
- [ ] PR is approved by 2 experts

### CODE REVIEW

File: app/views/layouts/application.html.haml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+!!!
+%html
+  %head
+    %meta{:content => "text/html; charset=UTF-8", "http-equiv" => "Content-Type"}/
```
Comment (Pavliv) [inline app/views/layouts/application.html.haml]: Please use new syntax style, instead "rocket"

File: app/views/layouts/application.html.haml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    %title ProDobro
+    = csrf_meta_tags
+    = stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload'
+    = javascript_include_tag 'application', 'data-turbolinks-track': 'reload'
+    %link{:href => "/images/hand.png", :rel => "shortcut icon", :type => "image/x-icon"}/
```
Comment (Pavliv) [inline app/views/layouts/application.html.haml]: Please delete end tags

File: app/assets/stylesheets/application.scss [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+}
+
+.navbar-nav .open .dropdown-menu {
+  cursor: pointer;
+}
+
+.navbar-nav .open .dropdown-menu {
+  li:hover {
+    color: #9edce5;
+  }
```
Comment (imysheniuk) [inline app/assets/stylesheets/application.scss]: Please, use ".navbar-nav .open .dropdown-menu" ones.
Use Nesting: https://www.sitepoint.com/sass-basics-nesting/

File: app/assets/stylesheets/application.scss [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+}
```
Comment (imysheniuk) [inline app/assets/stylesheets/application.scss]: Please, use ".navbar-nav .open .dropdown-menu" ones.
Use Nesting: https://www.sitepoint.com/sass-basics-nesting/

File: app/assets/stylesheets/application.scss [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+  .navbar-nav .open .dropdown-menu {
+    background-color: white;
+    cursor: pointer;
+    margin-right: 15px;
+    font-size: 14px;
+  }
+
+  .navbar-nav .open .dropdown-menu {
+    li:hover {
+      color: #9edce5;
```
Comment (imysheniuk) [inline app/assets/stylesheets/application.scss]: Same here (use nesting)

File: app/assets/stylesheets/application.scss [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+  font-size: 16px;
+}
+
+.navbar-nav .open .dropdown-menu {
+  cursor: pointer;
+}
+
+.navbar-nav .open .dropdown-menu {
+  li:hover {
+    color: #9edce5;
```
Comment (imysheniuk) [inline app/assets/stylesheets/application.scss]: If you use same color more than ones, you should use variable
https://htmlcolorcodes.com/tutorials/scss-color-variables/

File: app/views/layouts/application.html.haml (hunk 1 above)
Comment (imysheniuk) [reply app/views/layouts/application.html.haml]: Same in line 17
And also with all "href"

**Review by imysheniuk — APPROVED** [review]

**Review by Pavliv — APPROVED** [review]

**Pull request merged**